docs/source/changelog.rst

```diff
@@ -30,7 +34,9 @@ Changelog
         * Update dependencies (:pr:`738`, :pr:`741`, :pr:`747`)
 
     Thanks to the following people for contributing to this release:
-    :user:`jeff-hernandez`, :user:`chidauri`, :user:`christopherbunn`, :user:`kmax12`, :user:`MarcoGorelli`, :user:`angela97lin`, :user:`frances-h`, :user:`rwedge`, :user:`thehomebrewnerd`
+    :user:`angela97lin`, :user:`chidauri`, :user:`christopherbunn`,
```

Comment:
is there an extra line here?

Reply:
no, I think it's because the corresponding line in the old code wraps to a second line